specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/networkWatcher.json

```diff
@@ -728,6 +728,9 @@
         ],
         "operationId": "PacketCaptures_GetStatus",
         "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
```

Comment:
@fearthecowboy are we supporting these options now?

Reply:
@dsgouda I have confirmation of support in Python (already shipped part of the CLI), Ruby and Java. Whether C# supports it, I don't know :). Anyway, the C# runtime is doing "Location" by default, this is has no impact on C#, whether it's supported or not.